src/csharp/Microsoft.Spark/Services/ConfigurationService.cs

```diff
@@ -122,7 +121,9 @@ public int GetBackendPortNumber()
         }
 
         /// <summary>
-        /// Returns the port number used for communicating accumulator updates from JVM -> .NET driver
+        /// Returns the configured port number used for communicating accumulator updates from JVM -> .NET driver
+        /// This should only be used to retrieve startup configuration, the active server port should be retrieved from
+        /// SparkEnvironment.AccumulatorUpdateServer
```

Comment:
nit: We could get rid of the environment variable all together to simplify things. We're likely not to ever use it.

Reply:
Yeah makes sense, was initially thinking we could define the envvar in each of the microsoft-spark DotnetRunner `main()`'s but felt like overkill